src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,38 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private static final int SIZE = 10;
+    private final Object[] keys = new Object[SIZE];
+    private final Object[] values = new Object[SIZE];
+    private int countOfPairs = 0;
```

Comment:
```suggestion
    private int countOfPairs;
```
0 is the default value, so we don't need to set it explicitly